src/components/notifications-table/NotificationsTable.jsx

```diff
@@ -141,8 +145,50 @@ class NotificationsTable extends React.Component {
         },
     ];
 
+    componentDidMount = () => {
+        const { value } = this.state.statusFilter;
+        const items = [
+            {
+                id: "READY",
+                name: i18n.t("Ready"),
+            },
+            {
+                id: "RUNNING",
+                name: i18n.t("Running"),
+            },
+            {
+                id: "FAILURE",
+                name: i18n.t("Failure"),
+            },
+            {
+                id: "DONE",
+                name: i18n.t("Done"),
+            },
+        ];
+        this.setState({ statusFilter: { value, items } });
```

Comment:
All info is know, can we move it to the initial code?

Reply:
Yes! I mirrored the logic of the filters in BaseSyncConfigurator where the items are loaded async 😑